src/types/dto/attachments.dto.ts

```diff
@@ -2,7 +2,7 @@ import { boolean, z } from 'zod'
 import { FileTypes } from '@/types/interfaces'
 
 export const CreateAttachmentRequestSchema = z.object({
-  taskId: z.string().uuid(),
+  taskId: z.string(),
```

Comment:
Is there a reason .uuid() was removed here?

Reply:
createAttachmentRequestSchema is used in createTaskSlice for storing attachments on task creation. Initially while creating attachments on task creation, they dont have a taskId so it will be an empty string. Later on, while posting the attachments on db, taskId is appended from the created task. So, uuid was removed. Should i create another schema for posting attachments without task id for posting attachments on task creation?